src/Microsoft.Health.Fhir.Api/Controllers/AadSmartOnFhirProxyController.cs

```diff
@@ -189,19 +187,19 @@ public AadSmartOnFhirProxyController(IOptions<SecurityConfiguration> securityCon
             [FromQuery(Name = "error")] string error,
             [FromQuery(Name = "error_description")] string errorDescription)
         {
-            Uri redirectUrl = new Uri(Base64UrlEncoder.Decode(encodedRedirect));
+            var redirectUrl = new Uri(Base64UrlEncoder.Decode(encodedRedirect));
 
             if (!string.IsNullOrEmpty(error))
             {
-                return Redirect($"{redirectUrl.ToString()}?error={error}&error_description={errorDescription}");
+                return Redirect($"{redirectUrl}?error={error}&error_description={errorDescription}");
```

Comment:
Do error and errorDescription need to be URL encoded?

Reply:
@hansenms - Do you know the answer to this?